src/wizards/newsletters/views/tracking/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useEffect, useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { ActionCard, withWizardScreen } from '../../../../components/src';
+
+const apiPath = '/newspack/v1/wizard/newspack-newsletters/settings/tracking';
+
+export default withWizardScreen( () => {
+	const [ inFlight, setInFlight ] = useState( false );
+	const [ tracking, setTracking ] = useState( {} );
+
+	const fetchData = () => {
+		setInFlight( true );
+		apiFetch( { path: apiPath } )
+			.then( response => {
+				setTracking( response );
+			} )
+			.finally( () => {
+				setInFlight( false );
+			} )
+	};
+
+	const handleChange = type => async ( value ) => {
+		const newData = {
+			...tracking,
+			[ type ]: value,
+		};
+		setInFlight( true );
+		apiFetch( {
+			path: apiPath,
+			method: 'POST',
+			data: newData
+		} )
+			.then( () => {
+				setTracking( newData );
+			} )
+			.finally( () => {
+				setInFlight( false );
+			} );
+	};
+
+	useEffect( () => {
+		fetchData();
+	}, [] );
+
+	return (
+		<>
+			<ActionCard
```

Comment:
```suggestion
			<h1>{ __( 'Newsletter Tracking', 'newspack-plugin' ) }</h1>
			<ActionCard
```

Reply:
Same as above:

<img width="543" alt="image" src="https://github.com/user-attachments/assets/df9a665a-ecc9-4882-931b-117afe9037d7">

Reply:
Same here, I'd say "Tracking" is sufficient

Reply:
Added in ac26dbc1df32e3810831bcbf0211853694332724